lib/internal/bootstrap/loaders.js

```diff
@@ -270,29 +259,32 @@
       const cache = codeCacheHash[id] &&
         (codeCacheHash[id] === sourceHash[id]) ? codeCache[id] : undefined;
 
-      // (code, filename, lineOffset, columnOffset
-      // cachedData, produceCachedData, parsingContext)
-      const script = new ContextifyScript(
-        source, this.filename, 0, 0,
-        cache, false, undefined
+      const params = ['exports', 'require', 'module', 'process', 'internalBinding'];
```

Comment:
Now come to think of it, to reproduce the cache in the user land, we'll need to expose the params as well (otherwise we'll need to copy-paste)..Can you put this onto `NativeModule.parameters` (or something with a better name?)

Reply:
make sure you Object.freeze that

Reply:
@devsnek right. `ETOOMUCHMONKEYPATCHING`.

Reply:
Wait, `NativeModule` is accessible to userland?

Reply:
I guess it's not, now that I think about it. But still, better be safe than be sorry, I guess. 🤷‍♂️